resolver/contentResolver.go

```diff
@@ -0,0 +1,69 @@
+package resolver
+
+import (
+	"net/http"
+	"io/ioutil"
+	log "github.com/Sirupsen/logrus"
+	"encoding/json"
+	"github.com/Financial-Times/transactionid-utils-go"
+	"fmt"
+)
+
+type contentResolver struct {
+	tid string
+	contentResolverAppName string
```

Comment:
you don't have a config parameter for the `contentResolverAppName`. I'd get rid of it. You can determine what app was called by looking at the uri.